tests/pytests/unit/test_fileserver.py

```diff
@@ -76,7 +76,7 @@ def test_file_server_url_escape(tmp_path):
         "fileserver_backend": ["roots"],
         "extension_modules": "",
         "optimization_order": [
-            0,
+            0, 1
```

Comment:
Nice catch, updating the formatting here. 

I'll be honest that I don't totally understand why we need this. My limited understanding is that because we run the tests from a venv, it adds an additional level for the salt loader module. I didn't investigate the root cause, i.e. what exactly happens at the salt loader level when we run this from venv so I might be wrong here.